custom_components/signalrgb/config_flow.py

```
@@ -69,6 +69,12 @@ async def async_step_user(self, user_input: dict[str, Any] | None = None) -> Con
             await self.async_set_unique_id(f"{user_input[CONF_HOST]}:{user_input[CONF_PORT]}")
             self._abort_if_unique_id_configured()
             return self.async_create_entry(title=user_input[CONF_HOST], data=user_input)
+        finally:
+            if client is not None:
+                try:
+                    await client.aclose()
+                except Exception:  # pylint: disable=broad-except # noqa: BLE001
+                    pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)